metadata.tvdb.com/tvdb.xml

```diff
@@ -58,12 +58,18 @@
 			<RegExp input="$$1" output="\1" dest="5">
 				<expression noclean="1">&lt;Series&gt;.*?&lt;id&gt;$$2&lt;/id&gt;(.*)</expression>
 			</RegExp>
-			<RegExp input="$$5" output="&lt;plot&gt;\1&lt;/plot&gt;" dest="4">
-				<expression noclean="1">&lt;Overview&gt;([^&lt;]*)&lt;/Overview&gt;</expression>
-			</RegExp>
-			<RegExp input="$$2" output="&lt;id&gt;\1&lt;/id&gt;" dest="4+">
+			<RegExp input="$$5" output="&lt;uniqueid type=&quot;thetvdb&quot; default=&quot;true&quot;&gt;$$2&lt;/uniqueid&gt;" dest="4">
```

Comment:
The [Kodi documentation] shows the "key" for uniqueid as "tvdb". Ditto ratings, as well as "tmdb" rather than "themoviedb" in the other scrapers. The official documentation and the scrapers should agree, though I don't have a strong suggestion either way.

[Kodi documentation]: https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#ga97733fba546b3d217bf31d1701ebacb8